src/Nethermind/Nethermind.State/VerkleWorldState.cs

```diff
@@ -95,6 +95,11 @@ public VerkleWorldState(ExecutionWitness? executionWitness, Banderwagon root, IL
         _storageProvider = new VerkleStorageProvider(_tree, logManager);
     }
 
+    public bool ValuePresentInTree(Hash256 key)
```

Comment:
is there a better way to handle this? dont like exposing the internals of the tree to the worldState